homeassistant/components/husqvarna_automower/coordinator.py

```diff
@@ -172,7 +173,7 @@ async def client_listen(
             # Reset reconnect time after successful connection
             self.reconnect_time = DEFAULT_RECONNECT_TIME
             await automower_client.start_listening()
-        except HusqvarnaWSServerHandshakeError as err:
+        except (HusqvarnaWSServerHandshakeError, HusqvarnaWSClientError) as err:
```

Comment:
Should we catch the base class HusqvarnaAutomowerError instead?

Should we catch HusqvarnaWSServerHandshakeError?